solidity-plugin/src/main/java/org/sonarsource/solidity/SoliditySensor.java

```diff
@@ -82,25 +83,52 @@ public void analyzeFiles(SensorContext context, List<InputFile> files) {
     for (InputFile file : files) {
       String lastAnalyzedFile = file.toString();
       if (inSonarQube(context)) {
-        LOG.debug("Analyzing: " + lastAnalyzedFile);
-        getSyntaxHighlighting(context, file).save();
+        try {
+          LOG.debug("Analyzing: " + lastAnalyzedFile);
+          SolidityParser parser = Utils.returnParserUnitFromParsedFile(file.contents());
+          getSyntaxHighlighting(parser, context, file).save();
+          saveFileMeasures(context, computeMeasures(parser, fileLinesContextFactory.createFor(file), file), file);
+        } catch (IOException e) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Either remove or fill this block of code. ([squid:S00108](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AS00108))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=16&issues=AWQTGK75bXXTsx7O4tIV&open=AWQTGK75bXXTsx7O4tIV)

[ISSUE_KEY]: AWQTGK75bXXTsx7O4tIV